ui-kit/src/components/Modal/index.tsx

```diff
@@ -40,9 +40,12 @@ const Modal = ({ show, size = 'small', children, onOpen, onClose }: ModalProps)
     <div className={classnames('lubycon-modal')} tabIndex={-1} aria-hidden={true}>
       <ModalBackdrop onClick={handleBackdropClick} ref={backdropRef} />
       <ModalWindow size={size}>
-        {children.map((element) => {
-          return cloneElement(element, { key: generateID('lubycon-modal__children'), size: size });
-        })}
+        {Array.isArray(children)
+          ? children.map((element) =>
+              cloneElement(element, { size: size, key: generateID('lubycon-modal__children') })
+            )
+          : cloneElement(children, { size: size, key: generateID('lubycon-modal__children') })
+        }
```

Comment:
이거 그냥 `React.Children.map` 메소드 사용하면 `children`의 단일, 배열 여부에 상관없이 동일한 동작을 시킬 수 있슴다.

```suggestion
        {React.Children.map(children, child => {
          return cloneElement(child, { size: size, key: generateID('lubycon-modal__children') });
        })}
```

Reply:
그리고 children의 배열 여부를 검사할 때 `isArray`를 써도 되는데, [React.Children.only](https://ko.reactjs.org/docs/react-api.html#reactchildrenonly) 함수를 사용해도 됩니다!

```ts
const isOnly = (children: ReactNode) => {
  try {
    React.Children.only(children);
    return true;
  } catch {
    return false;
  }
}
```